src/style.css

```diff
@@ -59,6 +72,10 @@ h3 {
   margin: 10px;
 }
 
+.card-body {
+  padding-top: 0 !important;
+}
+
 p {
   font-size: 1rem;
   text-align: end;
```

Comment:
This style causes the following issue, i suggest you remove it.

![image](https://user-images.githubusercontent.com/53308506/152270325-2450e21a-004b-48b7-b101-d9b284dac984.png)